spark/v3.4/spark-extensions/src/main/scala/org/apache/spark/sql/execution/datasources/v2/CreateOrReplaceBranchExec.scala

```diff
@@ -32,6 +32,7 @@ case class CreateOrReplaceBranchExec(
                               ident: Identifier,
                               branch: String,
                               branchOptions: BranchOptions,
+                              createOrReplace: Boolean,
```

Comment:
The only think I feel is a little odd is that we have two booleans that are somewhat overlapping.  It feels like we should have `create` and `replace` and if both are set to `true`, you get `createOrReplace`.

Reply:
makes sense, I've updated the code